site/content/contribute/getting-started/contribution-checklist.md

```diff
@@ -16,7 +16,7 @@ Follow this checklist for submitting a pull request (PR):
 2. Your ticket is a help wanted GitHub issue for one of the Mattermost project you're contributing to.
```

Comment:
Unrelated to these changes, but should we take the opportunity to correct the URL in line 8 while we're in here? It didn't redirect properly when I tried to click it. Would replace `https://pre-release.mattermost.com/core/channels/tickets` with `https://community.mattermost.com/core/channels/tickets`. Or I can submit a separate PR for that change if it's best.

Reply:
Probably best to have it on separate PR for tracking purpose only.

Reply:
Makes sense, thanks, and done: https://github.com/mattermost/mattermost-developer-documentation/pull/208